guides/common/modules/proc_configuring-pulpcore-workers.adoc

```diff
@@ -0,0 +1,23 @@
+[id="Configuring_Pulpcore_Workers_{context}"]
+= Configuring Pulpcore Workers
+
+The pulpcore workers are the ones that fetch things from the content delivery network, process metadata etc. and the number of those will have an influence on the sync performance.
+
+Decreasing the number does not give any benefit from the sync speed point of view.
+Increasing the number gives a marginal benefit from the sync speed point of view, at the expense of almost doubling system load and increasing the PG load.
```

Comment:
Do you mean "PostgreSQL" by "PG"?

Reply:
yes.

Reply:
We haven't used PG for PostgreSQL so far. Did we? @maximiliankolb